modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css

```
@@ -82,3 +81,61 @@
 .congrats-information li img {
 	margin-left: 6px;
 }
+
+.right-column {
+	gap: 45px;
+}
+
+.outter-container {
+	display: none;
+}
+
+@media (max-width: 767.98px) {
+	.congrats-information {
+		width: 100%;
+	}
+
+	.right-column .download-print-share {
```

Change this selector to something like:
```suggestion
	.congrats-information .right-column .download-print-share {
```